docs/source/howto/installation.rst

```diff
@@ -18,7 +18,148 @@ Managing profiles
+By default, each AiiDA instance (each installation) will store associated profiles in this folder.
+A best practice is to always separate the profiles together with the code to which they belong.
+The typical approach is to place the configuration folder in the virtual environment itself and have it automatically selected whenever the environment is activated.
+
+The location of the AiiDA configuration folder, can be controlled with the ``AIIDA_PATH`` environment variable.
+This allows us to change the configuration folder automatically, by adding the following lines to the activation script of a virtual environment.
+For example, if the path of your virtual environment is ``/home/user/.virtualenvs/aiida``, add the following line:
+
+.. code:: bash
+
+    $ export AIIDA_PATH='/home/user/.virtualenvs/aiida'
+
+Make sure to reactivate the virtual environment, if it was already active, for the changes to take effect.
+
+.. note::
+
+   For ``conda``, create a directory structure ``etc/conda/activate.d`` in the root folder of your conda environment (e.g. ``/home/user/miniconda/envs/aiida``), and place a file ``aiida-init.sh`` in that folder which exports the ``AIIDA_PATH``.
+
+You can test that everything works by first echoing the environment variable with ``echo $AIIDA_PATH`` to confirm it prints the correct path.
+Finally, you can check that AiiDA know also properly realizes the new location for the configuration folder by calling ``verdi profile list``.
+This should display the current location of the configuration directory:
+
+.. code:: bash
+
+    Info: configuration folder: /home/user/.virtualenvs/aiida/.aiida
+    Critical: configuration file /home/user/.virtualenvs/aiida/.aiida/config.json does not exist
+
+The second line you will only see if you haven't yet setup a profile for this AiiDA instance.
+For information on setting up a profile, refer to :ref:`creating profiles<how-to:installation:profile>`.
+
+Besides a single path, the value of ``AIIDA_PATH`` can also be a colon-separated list of paths.
+AiiDA will go through each of the paths and check whether they contain a configuration directory, i.e., a folder with the name ``.aiida``.
+The first configuration directory that is encountered will be used as the configuration directory.
+If no configuration directory is found, one will be created in the last path that was considered.
+For example, the directory structure in your home folder ``~/`` might look like this::
+
+    .
+    ├── .aiida
+    └── project_a
+        ├── .aiida
+        └── subfolder
+
+If you leave the ``AIIDA_PATH`` variable unset, the default location ``~/.aiida`` will be used.
+However, if you set:
+
+.. code:: bash
+
+    $ export AIIDA_PATH='~/project_a:'
+
+the configuration directory ``~/project_a/.aiida`` will be used.
+
+.. warning::
+
+    If there was no ``.aiida`` directory in ``~/project_a``, AiiDA would have created it for you, so make sure to set the ``AIIDA_PATH`` correctly.
```

Comment:
I'm a bit confused by this warning. Why do I need to worry about this?

Would the following warning be correct?
```suggestion
    AiiDA will create the ``.aiida`` directory in ``~/project_a`` in case that it does not exist yet.
```

Or is the warning related to the fact that it will create the _complete path_ and not only the `.aiida/` leaf?

Reply:
The "problem" is that AiiDA will create the folder silently if it does not exist and will not raise an error. If a user, by accident, specifies the wrong path, the folder will be created in a different path then expected. Not that this will have grave consequences but it might be unexpected.